packages/webview_flutter/example/lib/main.dart

```diff
@@ -61,6 +61,11 @@ class WebViewExample extends StatelessWidget {
             print('allowing navigation to $request');
             return NavigationDecision.navigate;
           },
+          onPageFinished: (String url) {
+            Scaffold.of(context).showSnackBar(const SnackBar(
+              content: Text('Page finished loading!'),
```

Comment:
nit: this might be a little annoying? maybe just print? (probably worth printing the URL as well)